.claude/skills/agent-lifecycle/execute-lifecycle-hook.sh

```
@@ -1,573 +1,585 @@
+    echo "  spawn     --agent-id <id> --agent-type <type> [--acl-level <level>] [--name <name>]"
+    echo "  update    --agent-id <id> --confidence <score> --reasoning <text> [--phase <phase>] [--iteration <n>]"
+    echo "  complete  --agent-id <id> --confidence <score> [--output <text>] [--phase <phase>] [--iteration <n>]"
+    echo "  terminate --agent-id <id> [--reason <text>]"
+    echo "  status    --agent-id <id> [--limit <n>]"
+    echo ""
+    echo "Environment Variables:"
+    echo "  AGENT_LIFECYCLE_DB    Path to SQLite database (default: ./agent-lifecycle.db)"
+    echo ""
+    echo "Examples:"
+    echo "  $0 spawn --agent-id docker-1 --agent-type docker-specialist --acl-level 1"
+    echo "  $0 update --agent-id docker-1 --confidence 0.85 --reasoning 'Implementation complete'"
```

Shellcheck (0.11.0)

[warning] Did you forget to close this single quoted string?

(SC1078)